dev-client/src/theme.ts

```diff
@@ -56,6 +56,14 @@ export const theme = extendTheme({
       main: '#D32F2F',
       contrast: '#FFFFFF',
     },
+    alert: {
```

Comment:
suggestion: consider moving these to error.background, error.content and warning.content instead of creating a new alert section.

Reply:
I organized them as they are specified in the figma, but I do agree it makes more sense to just organize them under `error` and `warning` so will do that and note it in the figma.